epochStart/bootstrap/disabled/disabledHeaderIntegrityVerifier.go

```diff
@@ -0,0 +1,26 @@
+package disabled
+
+import (
```

Comment:
no need for a disabled component - you can use the real one.

Reply:
done

Reply:
not done - the file could be deleted

Reply:
done